jquery.flot.js

```diff
@@ -2536,14 +2536,19 @@ Licensed under the MIT license.
                 var points = series[i].datapoints.points;
                 ps = series[i].datapoints.pointsize;
 
+                // if the data is coming from positive -> negative, reverse the comparison
+                const comparer = points[points.length - ps] < points[0]
```

Comment:
This seems strange to me. Scatter data can easily fall through this new logic. I don't really understand why our interpolation logic would care whether the x-axis is reversed or not.

Reply:
You are correct that data where the x values are not ordered at all will not interpolate correctly... I am not sure whether there is a use case where this function would be used with unordered data however even prior to my changes this function would fail with unordered data

Reply:
Fair enough.